docs/cluster.md

```diff
@@ -0,0 +1,51 @@
+# KBS Cluster
+
+KBS provides a simple cluster defined by `docker-compose`, include itself, [Attestation Service](https://github.com/confidential-containers/attestation-service), [Reference Value Provider Service](https://github.com/confidential-containers/attestation-service/tree/main/bin/rvps) and [CoCo Keyprovider](https://github.com/confidential-containers/attestation-agent/tree/cc4e84c/sample_keyprovider)
+
+Users can use very simple command to:
+- launch KBS service.
+- encrypt images.
+
+## Architecture
+
+The basic architecture is
```

Comment:
I will remove this description and place the image directly.

BTW, you can use `<div align=center>![](./pictures/cluster.svg)` to place the picture in the center, which will be more beautiful :-)